app/src/actioncommands.cpp

```diff
@@ -417,6 +420,36 @@ void ActionCommands::flipSelectionY()
     mEditor->flipSelection(flipVertical);
 }
 
+void ActionCommands::selectAll()
```

Comment:
The logic for `selectAll` is in `ActionCommands` but the logic for `deselectAll` is in`Editor`, where's the logic in that? 😝  I feel like it belongs in `SelectionManager`, but unfortunately it need to access the current layer right now. I would be in favor of having both of them in the `Editor` for now, and maybe even removing these functions in `ActionCommands` entirely and connect directly to the functions in `Editor`.

Reply:
Ah.. yeah there's no logic in that, I started having it in actioncommands but then moved deselect to editor. Both will be placed in editor now but there will still be a proxy method in action commands, because that is where we make connections to UI that responds from UI.